Dialogs/PreferencesKeys.h

```diff
@@ -0,0 +1,46 @@
+#ifndef PREFERENCESKEYS_H
+#define PREFERENCESKEYS_H
+
+#include <QString>
+
+// key helpers to prevent typos and promote consistency
+inline QString preferencesKey() { return QStringLiteral("Preferences"); }
+
+inline QString generalKey() { return QStringLiteral("General"); }
+inline QString documentationURLKey() { return QStringLiteral("documentationURL"); }
+inline QString websiteURLKey() { return QStringLiteral("websiteURL"); }
+inline QString communityURLKey() { return QStringLiteral("communityURL"); }
+inline QString submitIssueURLKey() { return QStringLiteral("submitIssueURL"); }
+
+inline QString appearanceKey() { return QStringLiteral("Appearance"); }
+inline QString styleNameKey() { return QStringLiteral("styleName"); }
+
+#include <QSettings>
+
+// settings value helpers to prevent duplication of the defaults
+
+inline QString documentationURL() {
+  QSettings settings;
+  QString path = preferencesKey() + "/" + generalKey() + "/" + documentationURLKey();
+  return settings.value(path, "https://enigma-dev.org/docs/Wiki/Main_Page").toString();
+}
+
+inline QString websiteURL() {
+  QSettings settings;
+  QString path = preferencesKey() + "/" + generalKey() + "/" + websiteURLKey();

```

Comment:
I particularly like this path dynamic. Note: it's now "stylish" to overload `operator/` to do this. I don't know why and would not ask you to do that to this codebase...